fili-core/src/main/java/com/yahoo/bard/webservice/data/ResultSetSerializationProxy.java

```diff
@@ -96,14 +96,12 @@ public ResultSetSerializationProxy(ResultSet resultSet) {
      *
      * @return Schema components.
      */
-    private Map<String, Object> getSchemaComponents(Schema schema) {
+    private Map<String, Object> getSchemaComponents(ResultSetSchema schema) {
         Map<String, Object> schemaComponents = new HashMap<>();
 
-        String timeId = (schema instanceof ZonedSchema) ?
-                ((ZonedSchema) schema).getDateTimeZone().getID() :
-                SYSTEM_CONFIG.getStringProperty(SYSTEM_CONFIG.getPackageVariableName("timezone"), "UTC");
```

Comment:
Time zone is now riding along via the granularity rather than in the schema top level.